"Cannot find module 'npm'"

I use npm@3.1.2. Using grunt-aws-lambda gives me:

```
Loading "lambda_package.js" tasks...ERROR
>> Error: Cannot find module 'npm'
>>     at Function.Module._resolveFilename (module.js:336:15)
>>     at Function.Module._load (module.js:278:25)
>>     at Module.require (module.js:365:17)
>>     at require (module.js:384:17)
>>     at Object.module.exports (node_modules\grunt-aws-lambda\tasks\lambda_package.js:14:15)
```

Looks like `require('npm')` no longer works by default with npm 3.

`npm i npm@3.1.2` fixes it, so you can probably just include npm in the dependencies. No idea how you would elegantly pass the version of the globally installed npm though. A postinstall hook would work, I guess.

--- a/package.json
+++ b/package.json
@@ -32,15 +32,15 @@
         "mkdirp": "~0.5.0",
         "rimraf": "~2.2.8",
         "glob": "~4.3.0",
-        "aws-sdk": "~2.1.30"
+        "aws-sdk": "~2.1.30",
+        "npm": "^2.10.0"
     },
     "devDependencies": {
         "grunt-contrib-jshint": "^0.9.2",
         "grunt-contrib-clean": "^0.5.0",
         "grunt-contrib-nodeunit": "^0.3.3",
         "grunt": "~0.4.5",
-        "adm-zip": "~0.4.4",
-        "npm": "^2.10.0"
+        "adm-zip": "~0.4.4"
     },
     "peerDependencies": {
         "grunt": "~0.4.5"

